cmd/gitsign-credential-cache/README.md

```diff
@@ -0,0 +1,25 @@
+# gitsign-credential-cache
+
+`gitsign-credential-cache` is a helper binary that allows users to cache signing
+credentials. This can be helpful in situations where you need to perform
+multiple signing operations back to back.
+
+Credentials are stored in memory, and the cache is exposed via a Unix socket.
+Credentials stored in this cache are only as secure as the unix socket
+implementation on your OS - any user that can access the socket can access the
+data. When in doubt, we recommend **not** using the cache.
```

Comment:
Just to be clear, it's disabled by default right? Can we make that more explicit?

Reply:
Took another pass at this.